pygmt/figure.py

```diff
@@ -119,12 +120,24 @@ def _activate_figure(self) -> None:
         with Session() as lib:
             lib.call_module(module="figure", args=[self._name, fmt])
 
+    # TODO(PyGMT>=v0.20.0):  Remove the _preprocess method.
```

Comment:
Technically this `_preprocess` method is private so we could just remove it :fire: But if you think people are using it, we could do with 2 release cycles instead of 4.

Reply:
Yes, people are using it. 2 release cycles sounds good to me.